mrmustard/lab/abstract/state.py

```diff
@@ -526,15 +526,19 @@ def __getitem__(self, item) -> State:
         self._modes = item
         return self
 
-    def bargmann(self) -> Optional[tuple[ComplexMatrix, ComplexVector, complex]]:
-        r"""Returns the Bargmann representation of the state."""
+    def bargmann(self, numpy=False) -> Optional[tuple[ComplexMatrix, ComplexVector, complex]]:
```

Comment:
Is there any function that needs numpy=True? I don't see one in this PR. We have several mrmustard functions that return tensorflow objects, then usually it's on the user to convert it to numpy if they need for their purposes.

Reply:
Not in this PR no, but I keep needing to transform the triple into numpy, so I just added the option here.